src/proto/flwr/proto/control.proto

```diff
@@ -23,9 +23,6 @@ service Control {
   // Request to create a new run
   rpc CreateRun(CreateRunRequest) returns (CreateRunResponse) {}
 
-  // Get the status of a given run
-  rpc GetRunStatus(GetRunStatusRequest) returns (GetRunStatusResponse) {}
-
```

Comment:
Actually, I think we can remove the entire `control.proto`. But maybe not in this PR.